docs/src/test-reporters-js.md

```diff
@@ -429,6 +429,8 @@ npx playwright test --reporter="./myreporter/my-awesome-reporter.ts"
 * [Currents](https://www.npmjs.com/package/@currents/playwright)
 * [GitHub Actions Reporter](https://www.npmjs.com/package/@estruyf/github-actions-reporter)
 * [GitHub Pull Request Comment](https://github.com/marketplace/actions/playwright-report-comment)
+* [Mail Reporter](https://www.npmjs.com/package/playwright-mail-reporter)
```

Comment:
We are not comfortable with the package that requires a Resend API key.

Reply:
Fair feedback: I have updated the reporter and released a new version. The reporter now uses SMTP settings.